servicemanager/service/smplayservice.py

```diff
@@ -95,6 +98,10 @@ def start_from_binary(self):
         parent, _ = os.path.split(unzip_dir)
         force_pushdir(parent)
 
+        if "frontend" in self.service_data and self.service_data["frontend"]:
```

Comment:
shouldn't really be hardcoding service configs in code... is there another way to do this generically?